projects/canopy/src/lib/table/table/table.component.ts

```diff
@@ -57,6 +72,26 @@ export class LgTableComponent implements AfterContentChecked {
     }
   }
 
+  ngOnInit() {
+    if (!this._columnsBreakpoint) {
+      this.addColumnsBreakpoint(TableColumnLayoutBreakpoints.Medium);
+    }
+  }
+
+  private addColumnsBreakpoint(columnsBreakpoint: TableColumnLayoutBreakpoints) {
+    this.renderer.removeClass(
+      this.hostElement.nativeElement,
+      `lg-table--columns-${this._columnsBreakpoint}`,
+    );
+
+    this.renderer.addClass(
+      this.hostElement.nativeElement,
+      `lg-table--columns-${columnsBreakpoint}`,
+    );
+
+    this._columnsBreakpoint = columnsBreakpoint;
```

Comment:
Sorry to be fussy but I wondered if this line should go in the `setter`. I found it odd that the `getter` returned the private variable but the `setter` didn't set it. Which does mean the ngOnInit would need to change as well 😬

I think that pattern that we normally use the one below in the alert component. I haven't considered it too much before but I guess there is probably some additional functional calls

https://github.com/Legal-and-General/canopy/blob/master/projects/canopy/src/lib/alert/alert.component.ts

Reply:
Fixed